src/components/fields/SymbolSelector.js

```diff
@@ -377,10 +377,10 @@ class SymbolSelector extends Component {
 
     this.borderColor = Array.isArray(this.borderColor) ? COLORS.charcoal : this.borderColor;
 
-    if (this.props.is3D) {
-      this.symbolOptions = SYMBOLS.filter(option => {
-        return option.threeD;
-      });
+    if (this.props.container.type === 'scatter3d') {
+      this.symbolOptions = SYMBOLS.filter(option => option.threeD);
+    } else if (this.props.container.type === 'scattermapbox') {
+      this.symbolOptions = [SYMBOLS[0]];
```

Comment:
So when its scattermapbox, we only allow for the circle symbol? should we not allow at least some of the other variations, like open circle, etc?

Reply:
Yeah, actually all of them are supported alright, no need to treat it as special case. Fixed.